common/beerocks/bwl/include/bwl/mon_wlan_hal_types.h

```diff
@@ -64,29 +64,29 @@ struct SVapStats {
 };
 
 struct SStaStats {
-    float rx_rssi_watt;
-    uint8_t rx_rssi_watt_samples_cnt;
-    float rx_snr_watt;
-    uint8_t rx_snr_watt_samples_cnt;
+    float rx_rssi_watt = 0;
```

Comment:
Just like the comments the names of the vars from the old code are also a bit off, should probably be:
rx_rssi_mw/rx_rssi_mw_samples_cnt
and
rx_snr_ratio/rx_snr_ratio_samples_cnt